src/content/decentralization/index.md

```diff
@@ -0,0 +1,329 @@
+---
+title: What is a decentralized internet?
+description: An intro to decentralization on the web
+lang: en
+emoji: ":world_map:"
+sidebar: true
+sidebarDepth: 2
+summaryPoints:
+  [
+    "Decentralization is a design principle.",
+    "Ethereum is built on, benefits from, and pursues decentralization.",
+    "Decentralized and centralized systems can and do coexist.",
+  ]
+---
+
+## What's decentralization? {#what-is-decentralization}
+
+Decentralization is a way to create systems in which no individual or group can unilaterally make major decisions, especially not when it hurts the other participants. You can use this principle to build things in the fields of finance, architecture, art, communications, government, and — you guessed it — computer science.
+
+In basic terms: Nobody is in charge, but things still work. Rules without rulers.
```

Comment:
```suggestion
In basic terms: nobody is in charge, but things still work. Rules without rulers.
```

Reply:
This reminds me — which style guide does ethereum.org use? Probably should have asked before writing 😆